mythril/analysis/solver.py

```diff
@@ -62,6 +67,7 @@ def get_transaction_sequence(global_state, constraints):
     txs = {}
```

Comment:
Its not part of this pr, but I was wondering why the model used in the vulnerability check module isn't reused here